apps/azure-functions-api/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "azure-functions-api",
+  "version": "0.0.1",
+  "description": "Azure Function REST API Template",
+  "main": "./dist/main.js",
+  "license": "MIT",
+  "scripts": {
+    "prebuild": "shx rm -rf dist && npm run generate",
+    "postbuild": "dependency-check package.json --no-dev --missing ./dist/**/*.js",
+    "build": "yarn prebuild && tsc && yarn postbuild",
+    "watch": "tsc -w",
+    "extensions:install": "func extensions install",
+    "start": "dotenv -e .env func start --javascript",
+    "predev": "npm-run-all build",
+    "dev": "npm-run-all --parallel start watch",
+    "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx --cache",
+    "lint-api": "find ./api -name '*.yaml' -exec swagger-cli validate {} \\;",
+    "api:bundle": "bundle-api-spec -i api/internal.yaml.template -o api/internal.yaml -V $npm_package_version",
+    "pregenerate": "npm-run-all api:bundle",
+    "generate": "yarn pregenerate && shx rm -rf ./src/generated && npm-run-all generate:definitions-internal",
+    "generate:definitions-internal": "shx rm -rf ./src/generated/definitions/internal && shx mkdir -p ./src/generated/definitions/internal && gen-api-models --api-spec ./api/internal.yaml --no-strict --out-dir ./src/generated/definitions/internal ",
```

Comment:
can't we just have a single task  
`"generate:api-models": "gen-api-models --api-spec .api/internal.yaml --out-dir ./src/infra/http/models"` ?

Reply:
Removed useless script in d37bd1c